SQF/dayz_code/stringtable.xml

```diff
@@ -15891,7 +15892,7 @@
 		<Key ID="STR_EPOCH_10OZCOPPER_DESC">
 			<English>10oz Copper Bar</English>
 			<German>10oz Kupferbarren</German>
-			<Russian>Медный слиток (10 унций)</Russian>
+			<Russian>Медный слиток (10 унций)./Russian>
```

Comment:
@dreamforceinc 
1. Go [[here]](https://github.com/dreamforceinc/DayZ-Epoch/blob/master/SQF/dayz_code/stringtable.xml).
2. Click "Edit this file" (pen icon in the top right)
3. Make the changes
4. Save

It will automatically update here.

While you are at it, you may also want to shorten [these five translations](https://github.com/EpochModTeam/DayZ-Epoch/commit/1bd70a3d2dd56b409ea50f94b62acc33f16cf20f#diff-93280f000bc01a50bebae388204e3cedR13155) if possible, because they cut off in the right click menu as you mentioned on the forums.